CONTRIBUTING.md

```diff
@@ -320,7 +320,7 @@ push HEAD:refs/for/master`.
 *  Squash your various commits into one with `git rebase -i
    <initial commit>`, ensuring that the ultimate `Change-Id` in the
    commit is the one on the patch set on Gerrit. This workflow is
-   generally preferred.
+   generally preferred for more complex changes.
```

Comment:
hmm, I had no idea we have been recommending the squash/rebase approach until now... I mean I do use it when I have a series of (related or unrelated) patches that I need to reorganize/split/merge/rebase, but it's certainly more complex and overkill for the basic (and most frequent) cases.

I suggest swapping the order of the two bullet points so that the actually recommended and simpler workflow (amend) is listed first. Maybe even add some verbiage to the other bullet point to make it clear that you should just amend if you don't know what you're doing.